### PR TITLE
Update content_types.md to fix formatting issues for mkdocs

### PR DESCRIPTION
**Pull request by manez**
**GitHub Issue**: https://github.com/Islandora-CLAW/CLAW/issues/1098

# What does this Pull Request do?

Attempts to fix some formatting issues so that [this page](https://islandora-claw.github.io/CLAW/user-documentation/content_types/) looks better.

# What's new?

* Change tips to notes
* add titles to all notes
* `-` instead of `*` for bulleted lists.

# How should this be tested?

Implement that changes, rebuild the docs, and see if they look better now. 

# Additional Notes:
I still have no idea why the images are broken. I can't find anything in the formatting for this page that differs from other pages where images are showing up fine. The images are in the assets folder, they're the same filetype as images that do work. I am :confused:

# Interested parties
@dannylamb @whikloj @seth-shaw-unlv

### CODE REVIEW

File: docs/user-documentation/content_types.md [hunk 1]
```diff
@@ -133,36 +133,36 @@ The mapping itself consists of the `types`' and `fieldMappings` configurations.
...
+      - islandora: http://islandora.ca/CLAW/
+      - pcdm: http://pcdm.org/models#
+      - use: http://pcdm.org/use#
+      - iana: http://www.iana.org/assignments/relation/
+ - islandora_demo
+      - relators: http://id.loc.gov/vocabulary/relators/
+ - controlled_access_terms
+      - wgs84_pos: http://www.w3.org/2003/01/geo/wgs84_pos#
+      - org: https://www.w3.org/TR/vocab-org/#org:
+      - xs: http://www.w3.org/2001/XMLSchema#
```
Comment (whikloj) [inline docs/user-documentation/content_types.md]: This is not working, I think the hyphens are correct (as the below changes in Further Reading look correct) but here its a huge block of text. I'll play with it, but also you can install mkdocs on your machine and then in the CLAW directory run `mkdocs serve` which will build your docs and serve them on `localhost:8111` to check them

**Comment by manez** [comment]
Further Reading seems to have been working even with asterisks, so maybe it's the spacing. I'll install mkdocs and play around with it some more.

File: docs/user-documentation/content_types.md (hunk 1 above)
Comment (seth-shaw-unlv) [reply docs/user-documentation/content_types.md]: I'm sure this is my fault. There shouldn't be a leading space before the first-level items' hyphen and, [according to mkdocs](https://python-markdown.github.io/#differences), there should _only_ be 4 spaces before the sub-items, _not_ 6. 
```
- controlled_access_terms
    - wgs84_pos: http://www.w3.org/2003/01/geo/wgs84_pos#
    - org: https://www.w3.org/TR/vocab-org/#org:
    - xs: http://www.w3.org/2001/XMLSchema#
```

Comment (manez) [reply docs/user-documentation/content_types.md]: Updated the PR accordingly

**Pull request merged**